src/mongo/nodes.ts

```diff
@@ -196,6 +226,42 @@ export class MongoCollectionNode implements INode {
 	private _loadMoreNode: LoadMoreNode = new LoadMoreNode(this);
 	private _hasMore: boolean;
 	private _iterator: Cursor;
+	private _data: Object;
+
+	get data(): Object {
+		let docArray: Array<Object> = [];
+		for (let child of this._children) {
+			docArray.push(child.data);
+		}
+		this._data = docArray;
+		return this._data;
+	}
+
+	async update(data: any): Promise<any> {
+		if (!Array.isArray(data)) {
+			data = [data];
+		}
+		const operations = MongoCollectionNode.getBulkWriteUpdateOperations(data);
+		const result = await this.collection.bulkWrite(operations);
+		this._data = data;
+		return this._data;
+	}
+
+	static getBulkWriteUpdateOperations(data: any): any {
```

Comment:
static since it's also used by `MongoDummyNode`. Any suggestions for a better place to put this function?

Reply:
Leave it here and remove MongoDummyNode